assets/blockchains/ethereum/floki/info.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Floki",
+  "symbol": "FLOKI",
+  "status": "active",
+  "defaultVisibility": true,
+  "defaultOrdinalLevel": 100,
```

Comment:
Fix it.

Reply:
What should it exactly be? I've set the same as GT and STORJ.

Reply:
No specific position, but make them all different. E.g., 90, 95, 100, 105.